.changeset/six-wolves-admire.md

```diff
@@ -0,0 +1,22 @@
+---
+'@backstage/integration': minor
+---
+
+Added `AzureDevOpsCredentialsProvider` to support multiple Azure DevOps organizations and **deprecated** `AzureIntegrationConfig.credential` in favour of `AzureIntegrationConfig.credentials`. You can now use specific credentials for different Azure DevOps organizations by specifying the `organizations` field on a credential:
+
+```yaml
+integrations:
```

Comment:
I think we should include a PAT example as it's the most common setup right now

Reply:
Done 👍